app/src/main/java/com/sw_ss16/lc_app/ui/learning_center_one/StudyRoomDetailFragment.java

```diff
@@ -149,5 +147,6 @@ public static StudyRoomDetailFragment newInstance(String itemID) {
         return fragment;
     }
 
-    public StudyRoomDetailFragment() {}
+    public StudyRoomDetailFragment() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://www.codacy.com/app/lorenz-leitner/sw-ss16/file/2383261658/issues/source?bid=3226377&fileBranchId=3226702#l150)